pages/about/code-of-conduct.vue

```diff
@@ -38,20 +38,97 @@
                     >
                         {{ description }}
                     </p>
-                    <ol v-if="rule.list" class="list-decimal ml-5">
-                        <li
-                            v-for="(li, liIndex) in rule.list"
-                            :key="`coc-list-${liIndex}`"
-                        >
-                            {{ li }}
-                        </li>
-                    </ol>
-                    <p
-                        v-if="rule.addition"
-                        class="text-xs md:text-base leading-5 md:leading-8"
+                    <!-- 三大準則圖片 -->
+                    <div
+                        v-if="i == 0"
```

Comment:
**suggestion:** I think we could put a comment here to explain what is the meaning of `i == 0`. For example, the three principle images should be attached to the `Three Principles` section, which is the first item of the contents.

Reply:
ＯＫ